source/agora/consensus/EnrollmentManager.d

```diff
@@ -289,6 +294,10 @@ public class EnrollmentManager
 
             // consume pre-images
             this.cycle.populate(this.key_pair.v, true);
+
+            // save the count of 'populate' of `PreImageCycle`
+            this.setCycleIndex(
+                this.cycle.nonce * this.cycle.NumberOfCycles + this.cycle.index);
```

Comment:
Could you parenthesize these? `(this.cycle.nonce * this.cycle.NumberOfCycles) + this.cycle.index` or whichever precedence it is (it is not obvious).

Reply:
> Could you parenthesize these? (this.cycle.nonce * this.cycle.NumberOfCycles) + this.cycle.index or whichever precedence it is (it is not obvious).

Your suggestion could be more readable.